pkg/docker/config/config.go

```diff
@@ -72,6 +72,9 @@ func SetAuthentication(sys *types.SystemContext, registry, username, password st
 			}
 			logrus.Debugf("failed to authenticate with the kernel keyring, falling back to authfiles. %v", err)
 		}
+		fmt.Fprintf(os.Stderr, "%s\n", `WARNING! Your password will be stored unencrypted in authentication file
```

Comment:
```suggestion
		fmt.Fprintf(os.Stderr, "%s\n", `WARNING! Your password will be stored unencrypted in the authentication file.
```

Reply:
With the `'`, I'm not sure if there'll be a carriage return after this line or not.  If not, you need to make sure there's at least a space between it and the first word in the next line.